<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepthControlView</class>
 <widget class="QWidget" name="DepthControlView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DepthControlView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 44, 51);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
         </property>
         <property name="text">
          <string>点云显示</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SwitchButton" name="switch_clound_point" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>VideoMode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_video_mode">
       <property name="styleSheet">
        <string notr="true">QComboBox {
	color: rgb(255, 255, 255);
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
 
 
QComboBox:!editable, QComboBox::drop-down:editable {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
 
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
 
QComboBox::down-arrow {
    image: url(:/res/image/combobox_ico.png);
}
 
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView
{
     background-color: rgba(62,77,89);
}
 
QComboBox QAbstractItemView::item
{
    color: rgb(255, 255, 255);
    height: 24px;
}
 
QComboBox QAbstractItemView::item:selected
{	
    background-color: rgba(9,11,13);
}</string>
       </property>
       <item>
        <property name="text">
         <string>RAINBOW</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GRAY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HISTOGRAM</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cb_resolution_depth">
       <property name="styleSheet">
        <string notr="true">QComboBox {
	color: rgb(255, 255, 255);
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
 
 
QComboBox:!editable, QComboBox::drop-down:editable {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
 
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
 
QComboBox::down-arrow {
    image: url(:/res/image/combobox_ico.png);
}
 
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView
{
     background-color: rgba(62,77,89);
}
 
QComboBox QAbstractItemView::item
{
    color: rgb(255, 255, 255);
    height: 24px;
}
 
QComboBox QAbstractItemView::item:selected
{	
    background-color: rgba(9,11,13);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hSlider_bar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
       </property>
       <property name="text">
        <string>应用模式:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_filter_bin">
       <property name="styleSheet">
        <string notr="true">QComboBox {
	color: rgb(255, 255, 255);
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
 
 
QComboBox:!editable, QComboBox::drop-down:editable {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
       background-color: rgba(62,77,89);
       color: rgb(255, 255, 255);
}
 
QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
 
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
 
QComboBox::down-arrow {
    image: url(:/res/image/combobox_ico.png);
}
 
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView
{
     background-color: rgba(62,77,89);
}
 
QComboBox QAbstractItemView::item
{
    color: rgb(255, 255, 255);
    height: 24px;
}
 
QComboBox QAbstractItemView::item:selected
{	
    background-color: rgba(9,11,13);
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
       </property>
       <property name="text">
        <string>param1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="led_pointcloud_param1">
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);</string>
       </property>
       <property name="text">
        <string>80.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);
background-color: rgb(255, 255, 255，255);</string>
       </property>
       <property name="text">
        <string>param2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="led_pointcloud_param2">
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);</string>
       </property>
       <property name="text">
        <string>2.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>MaxDistance</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="le_depth_distance_value">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);</string>
       </property>
       <property name="text">
        <string>3000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_depth_distance_set">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(193, 211, 227);
background-color: rgb(62, 77, 89);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/res/image/write.png</normaloff>:/res/image/write.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="slider_depth_distance">
     <property name="maximum">
      <number>3000</number>
     </property>
     <property name="value">
      <number>3000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>MinDistance</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="le_depth_distance_value_min">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(195, 213, 229);</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_depth_distance_set_min">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(193, 211, 227);
background-color: rgb(62, 77, 89);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/res/image/write.png</normaloff>:/res/image/write.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="slider_depth_distance_min">
     <property name="maximum">
      <number>3000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header location="global">src\weight\switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
